tests/Event/ReadModel/JSONLD/EventLDProjectorTest.php

```diff
@@ -545,11 +545,15 @@ public function it_handles_new_events_with_multiple_timestamps()
                 '@type' => 'Event',
                 'startDate' => '2015-01-26T13:25:21+01:00',
                 'endDate' => '2015-01-27T13:25:21+01:00',
+                'eventStatus' => 'EventScheduled',
+                'eventStatusReason' => [],
             ],
             (object)[
                 '@type' => 'Event',
                 'startDate' => '2015-01-28T13:25:21+01:00',
                 'endDate' => '2015-01-29T13:25:21+01:00',
+                'eventStatus' => 'EventScheduled',
+                'eventStatusReason' => [],
```

Comment:
Hmm I think we might need to omit this if it's empty, instead of setting it to an empty array/object 🤔

Reply:
I've omitted it as part of https://github.com/cultuurnet/udb3-php/pull/482